### PR TITLE
handle if ser is all nan

### PR DESCRIPTION
**Pull request by itsjayway**
hello--please advise if this function should return a different format/value if this case arises

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added `MultiNestedTensor` ([#149](https://github.com/pyg-team/pytorch-frame/pull/149))
 
 ### Changed
+- Changed `compute_col_stats` ([#171](https://github.com/pyg-team/pytorch-frame/pull/171))
```
Comment (weihua916) [inline CHANGELOG.md:20]: Drop. Instead add `skip-changelog` flag

File: torch_frame/data/stats.py [hunk 2]
```diff
@@ -81,9 +81,9 @@ def compute_col_stats(
 
     if stype == torch_frame.numerical:
         ser = ser.mask(ser.isin([np.inf, -np.inf]), np.nan)
-    ser = ser.dropna()
 
     return {
-        stat_type: stat_type.compute(ser, sep)
+        stat_type: (np.nan if ser.isnull().all() else stat_type.compute(
+            ser.dropna(), sep))
```
Comment (weihua916) [inline torch_frame/data/stats.py:87]: Add test for the new case.

Comment (itsjayway) [reply torch_frame/data/stats.py]: got it, thanks @weihua916. 
image

File: torch_frame/data/stats.py [hunk 3]
```diff
@@ -81,9 +90,16 @@ def compute_col_stats(
 
     if stype == torch_frame.numerical:
         ser = ser.mask(ser.isin([np.inf, -np.inf]), np.nan)
-    ser = ser.dropna()
```
Comment (weihua916) [inline torch_frame/data/stats.py:84]: nit: 
```python
ser = ser.dropna()
if len(ser) > 0:
  stats = {
       stat_type: stat_type.compute(ser.dropna(), sep)
       for stat_type in StatType.stats_for_stype(stype)
  }
else:
   stats = {
       stat_type: _default_values[stat_type]
       for stat_type in StatType.stats_for_stype(stype)
   }
```

**Review by weihua916 — APPROVED** [review]

File: torch_frame/data/stats.py (hunk 2 above)
Comment (weihua916) [reply torch_frame/data/stats.py]: Thanks!

**Comment by weihua916** [comment]
Thank you!

**Pull request merged**